tests/secrets_ospdo.yaml

```diff
@@ -0,0 +1,61 @@
+# Command to login to OpenShift
+oc_login_command: |
+  oc login -u kubeadmin -p {{ admin_password }}
+
+tripleo_passwords: ~/tripleo-standalone-passwords.yaml #CUSTOMIZE_THIS
+
+# Adopted OSPD director operator env Baremetal fqdn
+adoption_ospdo_host: titan_xx.lab.eng.tlv2.redhat.com
```

Comment:
I wouldn't put values in here that are as env-specific as a hostname. But keeping the OSPDO example file is good i think.